library/newrelic-usage/dashboards/nr_usage_apof.json

```diff
@@ -0,0 +1,350 @@
+        },
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 1,
+            "row": 16,
+            "height": 3,
+            "width": 12
+          },
+          "title": "How many GBs have been ingested this month by ACCOUNT and TYPE?",
+          "rawConfiguration": {
+            "dataFormatters": [
+              {
+                "name": "Total GBs",
+                "precision": 2,
+                "type": "decimal"
+              }
+            ],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT filter(sum(GigabytesIngested), WHERE productLine = 'DataPlatform') AS 'Total GBs',filter(sum(GigabytesIngested), WHERE usageMetric ='InfraIntegrationBytes') AS 'Infra Integration', filter(sum(GigabytesIngested), WHERE usageMetric ='InfraProcessBytes') AS 'Infra Process', filter(sum(GigabytesIngested), WHERE usageMetric ='MetricsBytes') AS 'Metrics', filter(sum(GigabytesIngested), WHERE usageMetric ='InfraHostBytes') AS 'Infra Host', filter(sum(GigabytesIngested), WHERE usageMetric ='LoggingBytes') AS 'Logging', filter(sum(GigabytesIngested), WHERE usageMetric ='TracingBytes') AS 'Tracing', filter(sum(GigabytesIngested), WHERE usageMetric ='ApmEventsBytes') AS 'APM Events', filter(sum(GigabytesIngested), WHERE usageMetric ='ServerlessBytes') AS 'Serverless', filter(sum(GigabytesIngested), WHERE usageMetric ='BrowserEventsBytes') AS 'Browser Events', filter(sum(GigabytesIngested), WHERE usageMetric ='MobileEventsBytes') AS 'Mobile Events', filter(sum(GigabytesIngested), WHERE usageMetric ='CustomEventsBytes') AS 'Custom Events' FROM NrConsumption SINCE this month FACET consumingAccountId AS 'Account ID', consumingAccountName AS 'Account Name' LIMIT MAX"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 1,
+            "row": 19,
+            "height": 3,
+            "width": 4
+          },
+          "title": "How many GBs have I accrued this month?",
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from NrConsumption select sum(GigabytesIngested) as 'Total GBs Ingested this Month' where productLine = 'DataPlatform' SINCE this month"
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 5,
+            "row": 19,
+            "height": 3,
+            "width": 4
+          },
+          "title": "How many GBs have I accrued since my contract start date?",
```

Comment:
Consider
```suggestion
          "title": "How many GBs have I accrued since this year? (update query to reflect contract start date, if desired)",
```